lib/components/home/artwork_rail.js

```diff
@@ -50,6 +58,7 @@ export default Relay.createContainer(ArtworkRail, {
         results @include(if: $fetchContent) {
           __id
           title
```

Comment:
You no longer need the artwork `__id` and `title` here in this component, right?